-1 
Tested at: 4a3146aa32c39ca95d5138dbac3f33d2f465a762
 I found an error when building: 
 
 <pre>>> Building edm plugin tmp/slc6_amd64_gcc491/src/Alignment/CommonAlignmentMonitor/plugins/AlignmentMonitorTemplate/libAlignmentMonitorTemplate.so
tmp/slc6_amd64_gcc491/src/Alignment/CommonAlignmentMonitor/plugins/AlignmentStats/AlignmentStats.o: In function `AlignmentStats::analyze(edm::Event const&, edm::EventSetup const&)':
AlignmentStats.cc:(.text+0x388a): undefined reference to `Demangle::Demangle(char const*)'
tmp/slc6_amd64_gcc491/src/Alignment/CommonAlignmentMonitor/plugins/AlignmentStats/AlignmentStats.o: In function `std::string className<TrackingRecHit>(TrackingRecHit const&) [clone .isra.75]':
AlignmentStats.cc:(.text.unlikely+0x41): undefined reference to `Demangle::Demangle(char const*)'
collect2: error: ld returned 1 exit status
Leaving library rule at src/Alignment/MuonAlignment/plugins
gmake: *** [tmp/slc6_amd64_gcc491/src/Alignment/CommonAlignmentMonitor/plugins/AlignmentStats/libAlignmentStats.so] Error 1
>> Building edm plugin tmp/slc6_amd64_gcc491/src/Alignment/CommonAlignmentMonitor/plugins/AlignmentMonitorGeneric/libAlignmentMonitorGeneric.so
Leaving library rule at src/Alignment/CommonAlignmentMonitor/plugins
>> Building edm plugin tmp/slc6_amd64_gcc491/src/L1TriggerConfig/RPCTriggerConfig/test/DumpL1RPCHsbConfig/libDumpL1RPCHsbConfig.so
</pre> 
 you can see the results of the tests here: 
 https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-10299/4657/summary.html 

The following merge commits were also included on top of IB + this PR after doing git cms-merge-topic: 
https://github.com/cms-sw/cmssw/commit/227844018740bc6fc1f8a08f84c9e8fa3c711233
https://github.com/cms-sw/cmssw/commit/11ffca6009a5b207919cbc01ef604499a3160cd8
https://github.com/cms-sw/cmssw/commit/1d5698a8ee327d6df004ef3be6855606537f0f18
You can see more details here:
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-10299/4657/git-log-recent-commits
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-10299/4657/git-merge-result

--- a/Alignment/CommonAlignmentMonitor/plugins/BuildFile.xml
+++ b/Alignment/CommonAlignmentMonitor/plugins/BuildFile.xml
@@ -107,6 +107,7 @@
   <use   name="DataFormats/Alignment"/>
   <use   name="DataFormats/TrackReco"/>
   <use   name="DataFormats/TrackerRecHit2D"/>
+  <use   name="Utilities/General"/>
   <use   name="rootcore"/>
   <flags   EDM_PLUGIN="1"/>
 </library>

--- a/Alignment/HIPAlignmentAlgorithm/BuildFile.xml
+++ b/Alignment/HIPAlignmentAlgorithm/BuildFile.xml
@@ -9,4 +9,5 @@
 <use   name="Alignment/CommonAlignment"/>
 <use   name="Alignment/CommonAlignmentAlgorithm"/>
 <use   name="Alignment/TrackerAlignment"/>
+<use   name="Utilities/General"/>
 <flags   EDM_PLUGIN="1"/>

--- a/Alignment/TrackerAlignment/plugins/BuildFile.xml
+++ b/Alignment/TrackerAlignment/plugins/BuildFile.xml
@@ -11,5 +11,6 @@
   <use   name="DataFormats/TrackerRecHit2D"/>
   <use   name="DataFormats/Alignment"/>
   <use   name="RecoTracker/TransientTrackingRecHit"/>
+  <use   name="Utilities/General"/>
   <flags   EDM_PLUGIN="1"/>
 </library>

